addons/mk6mortar/CfgVehicles.hpp

```diff
@@ -34,6 +35,59 @@ class CfgVehicles {
                 discreteDistanceInitIndex = 0;
             };
         };
+        class ACE_Actions: ACE_Actions {
+            class GVAR(unloadMagazine) {
+                displayName = CSTRING(unloadMortar);
+                distance = 2;
+                condition = QUOTE(_this call FUNC(canUnloadMagazine));
+                statement = QUOTE([ARR_3(_target,_player,5)] call FUNC(unloadMagazineTimer));
+                icon = "";
+                selection = "usti hlavne";
+            };
+            class GVAR(LoadActions) {
+                displayName = CSTRING(loadMortar);
+                distance = 2;
+                condition = QUOTE([ARR_2(_target,_player)] call FUNC(canLoadMagazine));
+                statement = "";
+                icon = "";
+                selection = "usti hlavne";
+                class GVAR(loadMagazine_HE_Guided) {
+                    displayName = CSTRING(loadMagazine_HE_Guided);
+                    distance = 2;
+                    condition = QUOTE([ARR_3(_target,_player,'ACE_1Rnd_82mm_Mo_HE_Guided')] call FUNC(canLoadMagazine));
+                    statement = QUOTE([ARR_4(_target,_player,6,'ACE_1Rnd_82mm_Mo_HE_Guided')] call FUNC(loadMagazineTimer));
```

Comment:
I tried macroing it but I did something wrong. If you have a suggestion I'm all for it